webpack.config.js

```diff
@@ -4,7 +4,33 @@ const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const { UserscriptPlugin } = require('webpack-userscript');
 const packageJSON = require('./package.json');
+const process = require('node:process');
```

Comment:
unnecessary ``require`` statement. process is a global object in nodejs

Reply:
Node.js examples & best practices requires importing `node:process` manually (from Node.js doc).

Reply:
ok